docs/content/doc/setup/reverse-proxies.md

```diff
@@ -80,6 +80,22 @@ server {
 <b>NOTE:</b> If you change the max upload size in Vikunja's settings, you'll need to also change the <code>client_max_body_size</code> in the nginx proxy config.
 </div>
 
+## NGINX Proxy Manager (NPM)
+
+1. Create a Proxy Host within NPM and point it to whatever URL you plan to use. 
+2. Verify that the page will pull up in your browser. (Do not bother trying to log in. It won't work. Trust me.)
+3. Identify the container name for your NPM installation. e.g. `NGINX-PM`
+4. From the command line, enter sudo docker exec -it NGINX-PM /bin/bash and navigate to the proxy hosts folder where the `.conf` files are stashed. Probably `/data/nginx/proxy_host`.
```

Comment:
Are these mounted on the host? Otherwise this won't last a restart of npm.

Reply:
Yes, these are mounted on the host as part of the NPM container. (If you don't point NPM to a persistent folder, all of your proxy hosts will disappear with each restart.
If you Docker exec -it [NPM container name] /bin/bash/, you'll cd into the `/data/nginx/proxy_host` folder and be able to edit the `.conf` files.

Reply:
For your first note about which container, you're right. I missed that, but am not sure how you want me to handle the rewrite for that. I would change it to the following:
1. Create a Proxy Host for the Vikunja Frontend within NPM and point it to whatever URL you plan to use. The steps that follow will enable the Proxy Host to successfully navigate to the API (on port 3456).
2. Verify that the page will pull up in your browser. (Do not bother trying to log in. It won't work. Trust me.)
3. The next steps apply to the NPM container so you need to Identify the container name for your NPM installation. e.g. `NGINX-PM`

Reply:
>Yes, these are mounted on the host as part of the NPM container. (If you don't point NPM to a persistent folder, all of your proxy hosts will disappear with each restart.
> If you Docker exec -it [NPM container name] /bin/bash/, you'll cd into the /data/nginx/proxy_host folder and be able to edit the .conf files.

Ah okay, makes sense. I'd have edited them on the host but I guess that's a matter of preference. Should be fine the way you put it.

> For your first note about which container, you're right. I missed that, but am not sure how you want me to handle the rewrite for that. I would change it to the following:

I think that would work.